pyaedt/modeler/Primitives.py

```diff
@@ -1656,12 +1654,11 @@ def create_udp(self, udp_dll_name, udp_parameters_list, upd_library="syslib", na
         --------
         >>> my_udp = self.aedtapp.modeler.create_udp(udp_dll_name="RMxprt/ClawPoleCore",
         ...                                          udp_parameters_list=my_udpPairs,
-        ...                                          upd_library="syslib",
-        ...                                          udp_type="Solid")
+        ...                                          upd_library="syslib")
         <class 'pyaedt.modeler.Object3d.Object3d'>
 
         """
-        if ".dll" not in udp_dll_name:
+        if ".dll" not in udp_dll_name or ".py" not in udp_dll_name:
```

Comment:
I am not sure here, but should it be `and` instead of `or`
```suggestion
        if ".dll" not in udp_dll_name and ".py" not in udp_dll_name:
```